src/editor/components/components/AddLayerPanel/AddLayerPanel.component.jsx

```diff
@@ -17,10 +19,11 @@ import {
 } from './createLayerFunctions';
 import Events from '../../../lib/Events';
 
-const AddLayerPanel = ({ onClose, isAddLayerPanelOpen }) => {
+const AddLayerPanel = ({ onClose, isAddLayerPanelOpen, currentUser }) => {
```

Comment:
Please use 
`const { currentUser } = useAuthContext();`
instead of passing it as prop.